UnofficialSteamAuthenticator/App.xaml.cs

```diff
@@ -25,21 +25,21 @@ public sealed partial class App : Application
         public App()
         {
             Microsoft.ApplicationInsights.WindowsAppInitializer.InitializeAsync();
-            this.InitializeComponent();
-            this.Suspending += this.OnSuspending;
+            InitializeComponent();
+            Suspending += OnSuspending;
         }
 
         protected override void OnActivated(IActivatedEventArgs args)
         {
-            if (args.Kind == Windows.ApplicationModel.Activation.ActivationKind.Protocol)
+            if (args.Kind == ActivationKind.Protocol)
             {
                 var protocolArgs = (ProtocolActivatedEventArgs)args;
 
-                Frame rootFrame = Window.Current.Content as Frame;
+                Frame rootFrame = (Frame)Window.Current.Content;
```

Comment:
`Window.Current.Content` should **always** be of Type `Frame`, because you access properties of it later (`var page = rootFrame.Content;`)

Because of that, and because a suffix casting could potentially return `null`, a direct cast is used.